clients/src/main/java/org/apache/kafka/common/metrics/JmxReporter.java

```diff
@@ -69,20 +69,7 @@ public class JmxReporter implements MetricsReporter {
     private final Map<String, KafkaMbean> mbeans = new HashMap<>();
     private Predicate<String> mbeanPredicate = s -> true;
 
-    public JmxReporter() {
-        this("");
-    }
-
-    /**
-     * Create a JMX reporter that prefixes all metrics with the given string.
-     *  @deprecated Since 2.6.0. Use {@link JmxReporter#JmxReporter()}
-     *  Initialize JmxReporter with {@link JmxReporter#contextChange(MetricsContext)}
-     *  Populate prefix by adding _namespace/prefix key value pair to {@link MetricsContext}
-     */
-    @Deprecated
-    public JmxReporter(String prefix) {
-        this.prefix = prefix != null ? prefix : "";
-    }
+    public JmxReporter() {}
```

Comment:
Please initialize "prefix" to empty string and then remove this default constructor as well.

Reply:
@chia7712 I got it; I missed initializing "prefix" to empty string, thanks.